met-web/src/components/feedback/listing.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect } from 'react';
+import Grid from '@mui/material/Grid';
+import { Link } from 'react-router-dom';
+import SvgIcon from '@mui/material/SvgIcon';
+import { MetPageGridContainer, PrimaryButton } from 'components/common';
+import { Feedback } from 'models/feedback';
+import { useAppDispatch } from 'hooks';
+import { createDefaultPageInfo, HeadCell, PageInfo, PaginationOptions } from 'components/common/Table/types';
+import { ReactComponent as VeryDissatisfiedIcon } from 'assets/images/emojiVeryDissatisfied.svg';
+import { ReactComponent as DissatisfiedIcon } from 'assets/images/emojiDissatisfied.svg';
+import { ReactComponent as NeutralIcon } from 'assets/images/emojiNeutral.svg';
+import { ReactComponent as SatisfiedIcon } from 'assets/images/emojiSatisfied.svg';
+import { ReactComponent as VerySatisfiedIcon } from 'assets/images/emojiVerySatisfied.svg';
+import Stack from '@mui/material/Stack';
+import { openNotification } from 'services/notificationService/notificationSlice';
+import MetTable from 'components/common/Table';
+import { getFeedbacksPage } from 'services/feedbackService';
+
+const customRatings: {
```

Comment:
this is very similar to what we have in the feedback modal, we might be able to centralize this somewhere, no need to change it for this PR though